container/container.go

```diff
@@ -325,7 +325,7 @@ func (container *Container) SetupWorkingDirectory(rootIdentity idtools.Identity)
 }
 
 // GetResourcePath evaluates `path` in the scope of the container's BaseFS, with proper path
-// sanitisation. Symlinks are all scoped to the BaseFS of the container, as
+// sanitization. Symlinks are all scoped to the BaseFS of the container, as
```

Comment:
This one's okay either way (`sanitise`/`sanitize` are just the different british/american spellings) – but still fine to change!

Reply:
I agree, I use British English personally but consistency > personal preference IMO 😸